Assets/MixedRealityToolkit/SpatialMapping/Scripts/RemoteMapping/RemoteMeshTarget.cs

```diff
@@ -4,13 +4,18 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using UnityEngine;
+
 #if UNITY_EDITOR || UNITY_STANDALONE_WIN
 using System.Net;
 using System.Net.Sockets;
 #endif
-using UnityEngine;
 
-namespace MixedRealityToolkit.SpatialMapping.RemoteMapping
```

Comment:
Revert the namespace change for the Dev Branch, or retarget May or June Releases